server/test/services/export/AbstractExporterTest.java

```diff
@@ -967,6 +967,26 @@ FakeApplicationFiller answerFileUploadQuestion(
       return this;
     }
 
+    FakeApplicationFiller answerFileQuestionWithMultipleUpload(
+        QuestionModel question, String fileKey, int index) {
+      return answerFileQuestionWithMultipleUpload(question, null, fileKey, index);
+    }
+
+    FakeApplicationFiller answerFileQuestionWithMultipleUpload(
+        QuestionModel question, String repeatedEntityName, String fileKey, int index) {
```

Comment:
*nit, optional*: For other questions that take a list, the answerer method takes a list instead of an item and an index. Does this better matches the way the applicant data is filled in prod?